main/heart_stokes.cc

```diff
@@ -0,0 +1,205 @@
+#include "interfaces/stokes.h"
+#include "pidomus.h"
+
+#include "deal.II/base/numbers.h"
+
+#include "Teuchos_CommandLineProcessor.hpp"
+#include "Teuchos_GlobalMPISession.hpp"
+#include "Teuchos_oblackholestream.hpp"
+#include "Teuchos_StandardCatchMacros.hpp"
+#include "Teuchos_Version.hpp"
+
+#include "mpi.h"
+#include <iostream>
+#include <string>
+
+#include "elastic_problem.h"
+
+////////////////////////////////////////////////////////////////////////////////
+// Begin macros
+#define problem_Stokes(dim,spacedim,LAC) \
+  StokesInterface<dim,spacedim,LATrilinos> interface; \
+  piDoMUS<dim,spacedim,LAC> stokes ( \
+                                            "piDoMUS", \
+                                            interface); \
+  ParameterAcceptor::initialize(prm_file, pde_name+"_used.prm"); \
+  pidomus.lamdas.output_step = [&] (???)           \
```

Comment:
note that instead of `pidomus` you need to write `stokes`. You need to use the name of the object of the class piDoMUS, which in this case is `stokes`.